types/tx.go

```diff
@@ -129,7 +153,7 @@ func (tp TxProof) Validate() error {
 	return nil
 }
 
-func (tp *TxProof) VerifyProof() bool {
+func (tp TxProof) VerifyProof() bool {
```

Comment:
[question]
don't feel like you have to revert if not, but was this related to this PR? like, was this necessary for something to work?

Reply:
Nope. It was just a cosmetic to be consistent.